generamba/lib/generamba/module_generator.rb

```diff
@@ -5,88 +6,85 @@
 module Generamba
 
 	# Responsible for creating the whole code module using information from the CLI
-	class ModuleGenerator
+	class ModuleGenerator < Thor
```

Comment:
I think, perform this check in `gen_command.rb` right idea. And I noticed, that I don't sold this check, but only sold `yes` answer :( Then, I'll implementing this method in `xcodeproj_helper.rb` and move this check in `gen_command.rb`.